modules/covid_vaccine/app/jobs/covid_vaccine/expanded_submission_state_job.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module CovidVaccine
+  class ExpandedSubmissionStateJob
+    include Sidekiq::Worker
+    include SentryLogging
+    sidekiq_options retry: false
+
+    def perform
+      Rails.logger.info('Covid_Vaccine Expanded_Submission_State_Job Start')
+      states = CovidVaccine::V0::ExpandedRegistrationSubmission.group('state').count
+      state_data = {}
```

Comment:
Could you also add in a total count? Such as:

`total_submissions = CovidVaccine::V0::ExpandedRegistrationSubmission.count`

And maybe a check that sums up all of the state counts and returns whether that is equal to `total_submissions`?

I know, I know...but give a mouse a cookie...

Reply:
@batemapf added total_submissions, total_count_of_states and difference between them to Log output. 

output object like:
* `{:enrollment_pending=>1, :registered=>1, :received=>1, :enrollment_complete=>1, :enrollment_out_of_band=>1, :total_count_of_states=>5, :total_count_of_submissions=>5, :discrepancy_count=>0}`
* `{:enrollment_pending=>2, :registered=>1, :enrollment_out_of_band=>1, :total_count_of_states=>4, :total_count_of_submissions=>4, :discrepancy_count=>0}`